### ISSUE TITLE
🐛[BUG]: AIFS _fill_input bug

### ISSUE THREAD

**Issue by Corkman99** [NVIDIA/earth2studio#600]
### Version

0.11.0

### On which installation method(s) does this occur?

pip

### Describe the issue

Running AIFS I run into an error that stems from a difference in the shape of the input tensor and the corresponding CoordSystem object (105 vs 106 in the variable lengths). 

I have traced the error to be introduced in aifs.py, at the following lines (753, 754):

`to_remove = torch.arange(92, 101)
 out = torch.cat([out[:, :, :, :91, ...], out[:, :, :, 101:, ...]], dim=3)`

where static variables are removed using their indicies from a list. to_remove includes indicies 92, 93, ..., 100, while torch.cat cuts out indicies 91, 92, ..., 100. This seems to cause the off-by-one error I encounter. 

Is this intended?

**Comment by NickGeneva** [comment]
Hi @Corkman99 

Thanks for the report.

Can you elaborate on how you're encountering this error? I'm looking at our unit tests for this model and we do indeed use the input coordinates of the model to generate the input data, which suggests that it is consistent with respect to the number of variables.

The lines you mentioned (https://github.com/NVIDIA/earth2studio/blob/main/earth2studio/models/px/aifs.py#L754) do look suspicious, but I also know there is some manual manipulation to deal with animoi so it could be intended and an artifact of ecmwf's package.

i.e. should line 755 be updated to:

```
out = torch.cat([out[:, :, :, :92, ...], out[:, :, :, 101:, ...]], dim=3)
```

@loliverhennigh  can you take a quick look?

**Comment by NickGeneva** [comment]
Maybe we can check this bug in the following PR thats pulling the static inputs into the model wrapper internals

https://github.com/NVIDIA/earth2studio/pull/594

NVM this is for AIFS not AIFS ENS

**Comment by ViokingTung** [comment]
> Hi [@Corkman99](https://github.com/Corkman99)
> 
> Thanks for the report.
> 
> Can you elaborate on how you're encountering this error? I'm looking at our unit tests for this model and we do indeed use the input coordinates of the model to generate the input data, which suggests that it is consistent with respect to the number of variables.
> 
> The lines you mentioned (https://github.com/NVIDIA/earth2studio/blob/main/earth2studio/models/px/aifs.py#L754) do look suspicious, but I also know there is some manual manipulation to deal with animoi so it could be intended and an artifact of ecmwf's package.
> 
> i.e. should like 755 be updated to:
> 
> ```
> out = torch.cat([out[:, :, :, :92, ...], out[:, :, :, 102:, ...]], dim=3)
> ```
> 
> [@loliverhennigh](https://github.com/loliverhennigh) can you take a quick look?

Hi, Based on my analysis and test, it should be `out = torch.cat([out[:, :, :, :92, ...], out[:, :, :, 101:, ...]], dim=3)`

**Comment by loliverhennigh** [comment]
https://github.com/NVIDIA/earth2studio/pull/606
Ill try to get this merged tomorrow. This fixes it and make AIFS in sync with the newer AIFSENS implementation. Need to test tomorrow as having tech issues. Will ping for a review after that.
